src/itest/resources/features/insolvency-delta-error-path.feature

```diff
@@ -1,22 +1,15 @@
-Feature: Process insolvency delta information
+Feature: Process insolvency delta error path scenario
 
-  Scenario Outline: Processing insolvency delta information successfully
+  Scenario Outline: Processing invalid message
 
     Given Insolvency delta consumer service is running
-    When a "<message>" with "<companyNumber>" is published to the topic "insolvency-delta" and consumed
-    Then verify PUT method is called on insolvency-data-api service with body "<output>"
+    When a non-avro "<input>" is published and failed to process
+    Then the message should be moved to topic "insolvency-delta-insolvency-delta-consumer-invalid"
```

Comment:
Probably better to create a new "insolvency-delete-flows.feature" file with all delete scenarios, and existing ones (insolvency-delta-error-flows.feature, insolvency-delta-happy-path.feature) will remain untouched to represent three major EPICs we implemented in Insolvency